.github/workflows/pre-submit.cli.yml

```diff
@@ -1,5 +1,12 @@
-name: Pre submits
-on: [pull_request, workflow_dispatch]
+name: Pre submits cli
+on: 
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+  # Used in combination with schedule.cli.yml
+  # to avoid duplicating the test code.
```

Comment:
I'm not really understanding why the two files are necessary? It looks like you should be able to just include the `schedule.cli.yml` code here since it's only run when this worflow completes. Am I missing something?

Reply:
yes I can. I separated them to have the workflow filenames be "consistent" with when they run. Instead I could rename the workflow `pre-submit-schedule.cli.yml` and have the code all in one place. Let me know.

Reply:
`schedule.cli.yml` would run on pull_requests wouldn't it? I think it runs whenever the "Pre submits cli" workflow is completed regardless of the trigger that started it doesn't it?

Reply:
@ianlewis shall I change this or keep as-is?

Reply:
I'll go ahead and approve and we can adjust later if necessary.

Reply:
yes you're right it runs on PRs but does nothing except for schedule and workflows_dispatch events. (see `if`)